lib/chef/knife.rb

```diff
@@ -234,7 +234,7 @@ def self.load_deps
       dependency_loaders.each(&:call)
     end
 
-    OFFICIAL_PLUGINS = %w{ec2 rackspace windows openstack azure google linode push vcenter lpar}.freeze
+    OFFICIAL_PLUGINS = %w{linode lpar openstack push rackspace vcenter}.freeze
```

Comment:
linode is end of life so that should be removed. Push is part of Workstation and vcenter is in Workstation now.

Reply:
Okay @tas50 removed `linode`. I think for now we should keep `push` and `vcenter` till ChefDK EOL :)